src/js/mep-feature-universalgoogleanalytics.js

```diff
@@ -8,8 +8,8 @@
 $.extend(mejs.MepDefaults, {
 	googleAnalyticsTitle: '',
 	googleAnalyticsCategory: 'Videos',
-	googleAnalyticsEventPlay: 'Play',
-	googleAnalyticsEventPause: 'Pause',
+	googleAnalyticsEventPlay: 'mejs.play',
```

Comment:
Just add a new commit; rewriting a commit is difficult and potentially dangerous for the repo integrity